treedoc/utils.py

```diff
@@ -289,6 +289,71 @@ def ispackage(obj):
     return obj.__file__.endswith("__init__.py")
 
 
+def _get_name(param):
+    ''' Checks if signature.Parameter corresponds to *args or **kwargs type input.'''
+    if (param.kind == param.KEYWORD_ONLY or param.VAR_KEYWORD) and param.default is param.empty:
```

Comment:
If you mean to check against both `KEYWORD_ONLY` and `VAR_KEYWORD` you must write:
```python
if param.kind in (param.KEYWORD_ONLY, param.VAR_KEYWORD) and param.default is param.empty:
   ...
```